pwiz_aux/msrc/utility/vendor_api/Bruker/CompassData.hpp

```diff
@@ -331,6 +331,9 @@ struct PWIZ_API_DECL CompassData
     virtual InstrumentSource getInstrumentSource() const = 0;
     virtual std::string getAcquisitionSoftware() const = 0;
     virtual std::string getAcquisitionSoftwareVersion() const = 0;
+    virtual double getOneOverK0AcqRangeLower() const = 0;

```

Comment:
Add stubs in CompassData.cpp's CompassDataImpl similar to Baf2Sql.cpp.

Reply:
Right, missed that one